IHP/Breadcrumb/ViewFunctions.hs

```diff
@@ -0,0 +1,43 @@
+module IHP.Breadcrumb.ViewFunctions (
+    module IHP.Breadcrumb.Types,
+    renderBreadcrumb,
+    breadcrumbWithLink,
+    breadcrumbWithExternalLink,
+    breadcrumbWithoutLink,
+) where
+
+import IHP.Prelude
+import IHP.Breadcrumb.Types
+
+import IHP.ControllerSupport
+
+import Text.Blaze.Html (Html)
+
+import IHP.View.Types (BreadcrumbsView(..), styledBreadcrumb, styledBreadcrumbItem)
+import IHP.ViewSupport (theCSSFramework)
+import IHP.ControllerPrelude
+
+renderBreadcrumb :: (?context::ControllerContext) => [BreadcrumbItem] -> Html
+renderBreadcrumb breadcrumbItems = styledBreadcrumb theCSSFramework theCSSFramework breadcrumbItems breadcrumbsView
+        where
+            breadcrumbsView = BreadcrumbsView
+                { cssFramework = theCSSFramework
+                , breadcrumbItems = breadcrumbItemsRendered
+                }
+
+            breadcrumbItemsRendered =  [hsx|{forEach breadcrumbItems (styledBreadcrumbItem theCSSFramework theCSSFramework breadcrumbItems)}|]
+
+
+breadcrumbWithLink :: (Show controller, AutoRoute controller) => Html -> controller -> BreadcrumbItem
```

Comment:
What do you think about naming these function `breadcrumbLink` and `breadcrumbText`? IMO these look a little bit shorter to me and it feels nicer as they line up perfectly because they have the same string length :)

![image](https://user-images.githubusercontent.com/2072185/139534196-ce8b0ca5-ddad-4609-9cea-e43d5072d997.png)


![image](https://user-images.githubusercontent.com/2072185/139534208-718639fa-01ff-4484-9e90-3c5f905d2d68.png)